app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def new
+
+  end
+
+  def create
+    if session[:user]
+      flash[:notice] = 'Please log out before creating a new user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.